scripts/install

```diff
@@ -32,7 +32,6 @@ app=$YNH_APP_INSTANCE_NAME
 #=================================================
 ynh_script_progression --message="Validating installation parameters..."
 
-architecture=$(ynh_detect_arch)
```

Comment:
Can you explain why you remove this please?

Reply:
`ynh_detect_arch` has been deprecated in favor of `$YNH_ARCH` in line 26 https://github.com/YunoHost-Apps/kresus_ynh/blob/82a960414e8b25db9077eba5b42375d5e193dacb/scripts/install#L26

Reply:
Ah yes, I did not see that in the diff, thanks.